python/bind_operators.h

```diff
@@ -11,7 +11,7 @@
 namespace py = pybind11;
 
 template <class Other, class T, class... Ignored>
-void bind_comparison(pybind11::class_<T, Ignored...> &c) {
+void bind_inequality_to_operator(pybind11::class_<T, Ignored...> &c) {
```

Comment:
Why does this still exist? Shouldn't it be removed?

Reply:
So it's still there for `bind_mutable_view` `MaskConstView` etc, but those can maybe use the same comparison operations as we just defined?

Reply:
Ah, I see, it is so we can do `a.coords == b.coords`. Hmm, maybe that is good to keep?

Reply:
I did think that originally, but maybe that is also error prone (two similar mechanisms doing different things).